lib/mssql.js

```diff
@@ -65,7 +67,7 @@ MsSQL.prototype.connect = function(callback) {
   if (self.client) {
     return process.nextTick(callback);
   }
-  var connection = new mssql.Connection(this.settings, function(err) {
+  var connection = new mssql.Connection(this.clientConfig, function(err) {
```

Comment:
I think `this.connConfig` or `this.connectionConfig` would be a better property name, considering that the value is passed to a `Connection` constructor.